praw/__init__.py

```diff
@@ -1122,14 +1122,16 @@ def search(self, query, subreddit=None, sort=None, syntax=None,
         :param period: The time period of the results.
 
         The additional parameters are passed directly into
-        :meth:`.get_content`. Note: the `url` and `param` parameters cannot be
-        altered.
+        :meth:`.get_content`. Note: the `url` parameter cannot be altered.
 
         See https://www.reddit.com/wiki/search for more information on how to
         build a search query.
 
         """
         params = {'q': query}
+        if 'params' in kwargs:
```

Comment:
The "feature" parameter, for using the legacy_search method, is one that I can see myself using, but most definitely, the "after" parameter, "before", and "count" parameters, so one can get a certain "page" of the search. You can already technically do this with for example, the new "page" with `get_new(limit=100, params={'after' : 't3_examplecode'}) (as well as you the count/before) parameters; so when not being able to use it for search() I ended up with the query on /r/redditdev mentioned in the original pull request.  
Furthermore, this would allow developers to use any new parameters without having to wait for praw to catch up, given the hypothetical example that reddit changes to allow the new parameter `admin_only` which the valid values are True or False, one could simply pass that in `params`.
Is there any particular reason why you feel (if there is one) the change shouldn't be made (curious :P)?